TESTS/netsocket/udp/udpsocket_sendto_timeout.cpp

```diff
@@ -41,8 +41,11 @@ void UDPSOCKET_SENDTO_TIMEOUT()
     int sent = sock.sendto(udp_addr, tx_buffer, sizeof(tx_buffer));
     timer.stop();
     TEST_ASSERT_EQUAL(sizeof(tx_buffer), sent);
+#if MBED_CONF_TARGET_NETWORK_DEFAULT_INTERFACE_TYPE == CELLULAR
```

Comment:
This is always `0 == 0` as the macros are not defined and undefined macros are zero.

See https://github.com/ARMmbed/mbed-os/blob/master/features/netsocket/NetworkInterfaceDefaults.cpp#L116..L121

You need to define possible values before comparison.

Reply:
True. Timing on cellular network is not strict, so changed assert to print timeout in `tlssocket_recv_timeout.cpp` and `udpsocket_sendto_timeout.cpp` because those are failing frequently. I think that netsocket tests have many more far too strict timeouts that should be relaxed as well..?